src/controller/placeMaps.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import { StatusCode } from "../model/types/StatusCode";
+import {
+  filterHospitalPerDistance,
+  getHospitals,
+} from "../services/places.service";
+import { PlaceState } from "../model/types/PlaceState";
+import { HospitalDTOModel } from "../model/types/models/dto/HospitalDTOModel";
+
+/**
+ * Controlador para a rota /place/hospitals/maps.
+ * Espera os seguintes parâmetros de consulta:
+ * @param latitude: a latitude do usuário
+ * @param longitude: a longitude do usuário
+ * @param range: o alcance para buscar hospitais
+ * @returns PlaceState<Array<PlaceSearchHospitalsMap>>
+ * @throws NotFound
+ */
+async function placeSearchHospitalsMapsController(req: Request, res: Response) {
+  const latitude = Number(req.query.latitude);
+  const longitude = Number(req.query.longitude);
+  const range = Number(req.query.range);
+
+  if (isNaN(latitude) || isNaN(longitude)) {
+    res
+      .status(parseInt(StatusCode.notFound))
+      .send("Latitude and Longitude must be valid numbers.");
+    return;
+  }
+
+  const hospitalsList: PlaceState<Array<HospitalDTOModel>> =
```

Comment:
Acho que o tipo dessa variável já é inferida com a tipagem da função `getHospitals`. 
```suggestion
  const hospitalsList =
```